renderer/index.js

```diff
@@ -251,6 +251,35 @@ function addTorrentEvents (torrent) {
   }
 }
 
+function restoreSession () {
+  cfg.read(function (err, data) {
+    if (err) console.error(err)
+    state.saved = data
+    if (!state.saved.torrents) state.saved.torrents = []
```

Comment:
On initial read of config it's an empty object, so state.saved.torrents.forEach would throw an error.